### DISCUSSION TITLE
Horizontal page orientation

### DISCUSSION THREAD

**Discussion by Ilya-Chudinovskikh** [QuestPDF/QuestPDF#240] [General]
Hey everyone! Sorry in advance, if this is a stupid question, but I really couldn't find any related information in provided documentation and discussions here. My question is How can I rotate the whole page including text and tables inside to make it horizontal? The only way I found is to rotate elemets one by one, but I suppose it is not the best practise for my case.
![Без имени](https://user-images.githubusercontent.com/87187545/171399566-4a7ee039-8c8b-4c54-98d6-200d7344cd5d.png)

**Comment by lmingle** [comment]
It looks like what you need is .Landscape() on your page size.

ex.  page.Size(PageSizes.A7.Landscape());

**Reply by Ilya-Chudinovskikh** [reply]
Thanks a lot, this is exactly what I was looking for!

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
